common/crypto_utils/avalon_crypto_utils/crypto_utility.py

```diff
@@ -69,6 +70,21 @@ def base64_to_byte_array(b64_str):
 
 
 # -------------------------------------------------------------------------
+
+def generate_random_bytes(size):
+    """
+    Function to generate random initialization vector
```

Comment:
Not necessarily an iv. Could be generated for some other purpose as well.

Reply:
Agree, rephrased function comment